test/parallel/test-https-timeout-server.js

```diff
@@ -1,29 +1,29 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
```

Comment:
@GreenPioneer sorry to pester, this can also be moved after the `hasCrypto` check.